{
 "bad": [
  "errno",
  "errno.EEXIST",
  "os.path.dirname",
  "sys",
  "sys.version_info",
  "tokenize.NAME",
  "tokenize.NEWLINE",
  "tokenize.STRING",
  "unasync.__init__.Rule",
  "unasync.__init__.Token",
  "unasync.__init__._DEFAULT_RULE",
  "unasync.__init__._build_py",
  "unasync.__init__._get_tokens",
  "unasync.__init__._makedirs_existok",
  "unasync.__init__._tokenize",
  "unasync.__init__._untokenize",
  "unasync.__init__.unasync_files"
 ],
 "deps": {
  "__future__": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_2_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004",
   "python/conda-forge/linux-64/python-3.7.10-hffdb5ce_100_cpython"
  ],
  "collections": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_2_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004",
   "python/conda-forge/linux-64/python-3.7.10-hffdb5ce_100_cpython"
  ],
  "errno": [],
  "os": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_2_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004",
   "python/conda-forge/linux-64/python-3.7.10-hffdb5ce_100_cpython"
  ],
  "setuptools": [
   "setuptools/conda-forge/linux-64/setuptools-25.2.0-py35_0",
   "setuptools/conda-forge/linux-64/setuptools-28.0.0-py27_0",
   "setuptools/conda-forge/linux-64/setuptools-36.6.0-py36_0",
   "setuptools/conda-forge/linux-64/setuptools-40.6.2-py36_0",
   "setuptools/conda-forge/linux-64/setuptools-41.2.0-py27_0",
   "setuptools/conda-forge/linux-64/setuptools-46.4.0-py37hc8dfbb8_0"
  ],
  "sys": [],
  "tokenize": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_2_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004"
  ],
  "unasync": []
 }
}